addons/y2038/y2038.py

```diff
@@ -190,9 +190,8 @@ def reportTokDiag(template, cfg, token, severity, msg):
 def find_dump_files(paths):
     dumpfiles = []
     for path in paths:
-        if path.endswith('.dump'):
-            if path not in dumpfiles:
-                dumpfiles.append(path)
+        if path.endswith('.dump') and path not in dumpfiles:
```

Comment:
behaviour change if path ends with .dump and path is in dumpfiles. Then the condition is true and the else is executed.